addons/rating/views/rating_templates.xml

```diff
@@ -3,15 +3,19 @@
         <!-- External page : thanks message -->
         <template id="rating_external_page_view" name="Thanks for your Feedback">
             <t t-call="web.frontend_layout">
-                <div class="container pt-5">
-                    <div class="oe_structure" id="oe_structure_rating_header_thanks_you"/>
-                    <div class="text-center mt-5 pt-5">
-                        <h3 class="d-inline">Thank you, we appreciate your feedback!</h3>
+                <div class="o_rating_page_view d-flex flex-column justify-content-center">
+                    <div class="container py-5">
+                        <div class="oe_structure" id="oe_structure_rating_header_thanks_you"/>
+                        <div class="text-center mt-4 pb-2">
+                            <h3 class="d-inline">Thank you for your feedback!</h3>
+                        </div>
+                        <div class="text-center mb-4">
```

Comment:
Same here, would see if possible to get rid of the div to only keep the link tag.